var/spack/repos/builtin/packages/spades/package.py

```diff
@@ -29,3 +29,7 @@ class Spades(CMakePackage):
     conflicts('%gcc@7.1.0:', when='@:3.10.1')
 
     root_cmakelists_dir = "src"
+
+    def url_for_version(self, version):
+        url = "https://github.com/ablab/spades/releases/download/v{0}/SPAdes-{1}.tar.gz"
+        return url.format(version, version)
```

Comment:
```suggestion
```

I thought you removed old versions because they were unavailable on Github and had to be fetched from cap.spbu.ru. If they are on Github, no need for a custom `url_for_version`.